platform/darwin/scripts/style-spec-overrides-v8.json

```diff
@@ -1,4 +1,28 @@
 {
+  "layer": {
+    "type": {
+      "values": {
+        "fill": {
+          "doc": "An `MGLFillStyleLayer` is a style layer that renders one or more filled (and optionally stroked) polygons on the map.\n\nUse a fill style layer to configure the visual appearance of polygon or multipolygon features in vector tiles loaded by an `MGLVectorSource` object or `MGLPolygon`, `MGLPolygonFeature`, `MGLMultiPolygon`, or `MGLMultiPolygonFeature` instances in an `MGLShapeSource` object."
```

Comment:
Once fill extrusion layers land on the master branch, we’ll need to provide a similar documentation string for the `fill-extrusion` type.

/cc @boundsj